plugins/generator-1.19.2/forge-1.19.2/templates/dimension/portaltrigger.java.ftl

```diff
@@ -1,7 +1,7 @@
 <#--
  # MCreator (https://mcreator.net/)
  # Copyright (C) 2012-2020, Pylo
- # Copyright (C) 2020-2022, Pylo, opensource contributors
+ # Copyright (C) 2020-2023, Pylo, opensource contributors

```

Comment:
Why did you change it?

Reply:
Well I edited the file so copyright year can be updated to this year, can't it be?

Reply:
As far as I know, copyright years are only changed when creating the file/making a new generator. We never changed them before (from what I know). Also, if you change this file, you should also change the other files you changed for consistency as currently, this is the only file with a changed copyright year.

Reply:
Generally, if changes are done to the file, the copyright can be updated, but it is not really something that would be mandatory or needed